.travis.yml

```diff
@@ -7,6 +7,11 @@ jdk:
 install: true
 addons:
   srcclr: true
+env:
+  - optimizely_default_parser=com.google.gson.Gson
```

Comment:
We should use the enums here to make it clear there are a set number of supported parsers.